guides/v3.7.0/reference/accessibility.md

```diff
@@ -0,0 +1,96 @@
+The purpose of this guide is to give every Ember developer a starting point for making your Ember application minimally accessible. As accessibility can be a complex issue, it should be noted that this guide only covers a some of the basics, and should not be considered a comprehensive guide. 
+
+## Screen Reader and Browser Combinations
```

Comment:
I am wondering whether this should be the first section or not. If someone only reads this far, have they received the most important info, or would you rather other parts take precedence?